app/controllers/education_infos_controller.rb

```diff
@@ -1,22 +1,21 @@
 # frozen_string_literal: true
 
 class EducationInfosController < ApplicationController
-    def new
-    end
-    
-    def create
-        @education = Education.where(user_id: current_user).first_or_initialize
-        if(@education.update(education_params))
-            #Move to Work History Page
-        else
-            render 'new'
-        end
+  def new; end
+
+  def create
+    @education = Education.where(user_id: current_user).first_or_initialize
```

Comment:
@AkshayPashnathSuresh 
try `find_or_initialize` method also if you are interested. 
no of queries run to fetch result by `where` is way more than `find`.